### PR TITLE
ADD: Search for Wallet in Home screen (iOS)

### PR DESCRIPTION
**Pull request by marcosrdz**
Screenshot 2024-01-20 at 2 04 18 PM

Screenshot 2024-01-20 at 2 04 11 PM

### CODE REVIEW

File: screen/wallets/list.js [hunk 1]
```diff
@@ -51,6 +51,13 @@ const WalletsList = () => {
   const walletsCount = useRef(wallets.length);
   const walletActionButtonsRef = useRef();
 
+  // Header Search Bar for Wallets
```
Comment (Overtorment) [inline screen/wallets/list.js:54]: while youre at it, convert to ts

File: react-native.config.js [hunk 2]
```diff
@@ -9,4 +9,3 @@ module.exports = {
     },
   },
 };
-
```
Comment (Overtorment) [inline react-native.config.js:12]: bad craftsmanship.

use `git add -p`

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/vqewf4a2eaigd4hy22g3chwpti?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/216)

**Review by Overtorment — CHANGES_REQUESTED** [review]
when the search bar would become visible?

on pull down?

rememer btw when we had idea that on pull down you can see summary value of all your wallets? i like both ideas

**Comment by Overtorment** [comment]
> when the search bar would become visible?
> 
> on pull down?
> 
> rememer btw when we had idea that on pull down you can see summary value of all your wallets? i like both ideas

and btw, what if we wont limit search to wallet name only? search tx labels and amounts as well. say, i remember i made a tx 0f 0.666 and i want to find whichwallet, so i sype in search '666'. wqould be pretty cool

**Comment by marcosrdz** [comment]
> when the search bar would become visible?
> 
> 
> 
> on pull down?
> 
> 
> 
> rememer btw when we had idea that on pull down you can see summary value of all your wallets? i like both ideas

Yup on pull down. standard platform behavior

**Comment by Overtorment** [comment]
deepscan bitchin btw

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/2owagops5l5mxynyvhvfaq6a64?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/217)

**Comment by marcosrdz** [comment]
> > when the search bar would become visible?
> 
> > 
> 
> > on pull down?
> 
> > 
> 
> > rememer btw when we had idea that on pull down you can see summary value of all your wallets? i like both ideas
> 
> 
> 
> and btw, what if we wont limit search to wallet name only? search tx labels and amounts as well. say, i remember i made a tx 0f 0.666 and i want to find whichwallet, so i sype in search '666'. wqould be pretty cool

Ill write enhancements in another PR

**Comment by marcosrdz** [comment]
> deepscan bitchin btw

Fixed

File: screen/wallets/list.js (hunk 1 above)
Comment (marcosrdz) [reply screen/wallets/list.js:54]: That's plenty for this file. I can do that in another PR

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/4mxd74rn3ayju4tgkgj62gcqx4?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/219)

**Comment by marcosrdz** [comment]
> when the search bar would become visible?
> 
> 
> 
> on pull down?
> 
> 
> 
> rememer btw when we had idea that on pull down you can see summary value of all your wallets? i like both ideas

Valuable info should be displayed all the time. Pull down is for "tools" such as search, shortcuts or refresh.

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

File: react-native.config.js (hunk 2 above)
Comment (Overtorment) [reply react-native.config.js:12]: hola pendejo

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**